packages/core-modules/test/contracts/modules/CoreElectionModule.test.js

```diff
@@ -96,4 +96,30 @@ describe('CoreElectionModule', () => {
       });
     });
   });
+
+  describe('when the user self nominates', () => {
+    it('sets the value & unsets the value', async () => {
```

Comment:
np it can be splitted in different describes to identify where it fails if it fails

Reply:
Done!